spec/mongoid/slug_spec.rb

```diff
@@ -471,6 +471,47 @@ module Mongoid
       end
     end
 
+    context 'when block is configured globally' do
+      before do
+        Mongoid::Slug.config do |c|
+          c.slug = proc{|cur_obj|
```

Comment:
Sounds convoluted for a test. Maybe something very simple that uses a `SecureRandom.uuid` to be unique?